### PR TITLE
#3 Write a section about migrating from Adapters

### PR DESCRIPTION
**Pull request by Baltazore**

### CODE REVIEW

File: translations/adapters/path-for-type/en-us.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+title: Use request builders instead of pathForType
+description: >
+  To modify URL for request you can use '<code>'resourcePath'</code>' option for every request builder. Default configuration for '<code>'JSON:API'</code>', '<code>'REST'</code>', and '<code>'ActiveRecord'</code>' builders will be provided by EmberData.
+  <br />
+  We recommend creating you own utility file with request builders that suites your backend needs.
```
Comment (acorncom) [inline translations/adapters/path-for-type/en-us.yaml]:
```suggestion
  We recommend creating your own utility file with request builders that suite your backend's needs.
```

File: translations/adapters/path-for-type/en-us.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+title: Use request builders instead of pathForType
+description: >
+  To modify URL for request you can use '<code>'resourcePath'</code>' option for every request builder. Default configuration for '<code>'JSON:API'</code>', '<code>'REST'</code>', and '<code>'ActiveRecord'</code>' builders will be provided by EmberData.
```
Comment (acorncom) [inline translations/adapters/path-for-type/en-us.yaml]:
```suggestion
  To modify a URL for a request you can use the '<code>'resourcePath'</code>' option for every request builder. Default configuration for '<code>'JSON:API'</code>', '<code>'REST'</code>', and '<code>'ActiveRecord'</code>' builders will be provided by EmberData.
```

File: translations/adapters/host-and-namespace/en-us.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+title: Default Host and Namespace
+description: >
+  To set default '<code>'host'</code>' and '<code>'namespace'</code>' for requests, you can use '<code>'setBuildURLConfig'</code>' method of '<code>'@ember-data/request-utils'</code>' package.
+  <br />
+  This is module-level function, so you can do it anywhere in application, i.e. in your '<code>'services/request-manager.js'</code>' or '<code>'services/store.js'</code>' file.
```
Comment (acorncom) [inline translations/adapters/host-and-namespace/en-us.yaml]:
```suggestion
  To set the default '<code>'host'</code>' and '<code>'namespace'</code>' for requests, you can use the '<code>'setBuildURLConfig'</code>' method of '<code>'@ember-data/request-utils'</code>' package.
  <br />
  This is a module-level function, so you can do it anywhere in your application, i.e. in your '<code>'services/request-manager.js'</code>' or '<code>'services/store.js'</code>' file.
```

File: translations/adapters/general/en-us.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+title: Adapters in general
+description: |
+  In order to provide migration support for Adapter and Serializer, a '<code>'LegacyNetworkHandler'</code>' would be provided. This handler would take a request and convert it into the older form, calling the appropriate Adapter and Serializer methods. If no adapter exists for the type (including no application adapter), this handler would call '<code>'next'</code>'. In this manner an app can incrementally migrate request-handling to this new paradigm on a per-type basis as desired.
+  <br />
+  The package '<code>'ember-data'</code>' would automatically configure this handler. If not using '<code>'ember-data'</code>' this configuration would need to be done explicitly.
+  <br />
+  We intend to support this handler through at least the 5.x series, not deprecating it's usage before 6.0.
+  <br />
+  Similarly, the methods '<code>'adapterFor'</code>' and '<code>'serializerFor'</code>' will not be deprecated until at least 6.0; however, it should no longer be assumed that an application has an adapter or serializer at all.
```
Comment (acorncom) [inline translations/adapters/general/en-us.yaml]:
```suggestion
  In order to provide migration support for Adapters and Serializers, a '<code>'LegacyNetworkHandler'</code>' is provided. This handler takes a request and converts it into the older form, calling the appropriate Adapter and Serializer methods. If no adapter exists for the type (including no application adapter), this handler calls '<code>'next'</code>'. In this manner an app can incrementally migrate request-handling to this new paradigm on a per-type basis as desired.
  <br />
  The package '<code>'ember-data'</code>' automatically configures this handler. If not using the '<code>'ember-data'</code>' package, this configuration needs to be done explicitly.
  <br />
  We intend to support this handler through at least the 5.x series -- not deprecating its usage before 6.0.
  <br />
  Similarly, the methods '<code>'adapterFor'</code>' and '<code>'serializerFor'</code>' will not be deprecated until at least 6.0; however, it should no longer be assumed that an application has an adapter or serializer at all.
```

**Review by acorncom — COMMENTED** [review]
@Baltazore looks good here, a few tweaks to the verbiage. Thanks for working on this

**Comment by Baltazore** [comment]
@acorncom thanks for review, please don't merge it before @runspired review

**Comment by Baltazore** [comment]
closed in favor of #13

**Pull request closed without merging**